wrappers/cabal

```diff
@@ -10,5 +10,5 @@ if [ "$1" == "--interactive" ]; then
     out "$arg"
   done
 else
-  ghc "$@"
+  ${HIE_BIOS_GHC} "$@"
```

Comment:
Does this work if there's a space in the path?

Reply:
Fair, we could also set a default variable right here, so cabal doesnt have to if not necessary.